sofy_sofy/README.md

```diff
@@ -0,0 +1,52 @@
+# Sofy
+
+## Overview
+
+Sofy is a no-code platform for creating automated tests on  mobile Apps. Users can integrated with their CI/CD pipelines to execute tests on real devices and view the results of their functional tests along with performance metrics.
+
+With this integration, you have deeper insight into your testing process by visualizing key metrics and trends such as load time, network, memory utilization and CPU. The out-of-the-box dashboard provides real-time visibility into your SOFY test results, enabling you to monitor and analyze performance over time and make data-driven decisions to improve overall software quality.
```

Comment:
Is the standard to use Sofy or SOFY? Please ensure that the capitalization is consistent throughout the tile.